service/github/githubutils/utils_test.go

```diff
@@ -11,7 +11,7 @@ func TestLinkedMarkdownDiagnostic(t *testing.T) {
 	tests := []struct {
 		owner, repo, sha string
 		d                *rdf.Diagnostic
-		serverUrl	     string
+		serverUrl        string
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>struct field serverUrl should be serverURL